editor/src/templates/preview/preview.tsx

```diff
@@ -280,8 +279,6 @@ const initPreview = () => {
       }
     }
 
-    injectTwind(projectContents, requireFn)
```

Comment:
shouldn't we replace this with something pointing at the tailwind compiler? 
does the preview.tsx still work at all? if not, we should probably remove this file instead of letting it slowly regress

Reply:
https://github.com/concrete-utopia/utopia/pull/6401/commits/ef7ec9c60bf04122b82ff35739183678928e460e